values.yaml

```diff
@@ -276,7 +276,7 @@ pingProber:
     pullPolicy: IfNotPresent
 
 pingExporter:
-  enabled: true
+  enabled: false
```

Comment:
@KilianDargel semantic change here. while in general i support disabling a feature per default: this here sneaks in and might break things.

Reply:
+1 for @mgumz comment -- you should not change the default here.